### PR TITLE
Intro and background edits

### PR DESCRIPTION
**Pull request by mdhaber**
Tweaks to intro intended to make language more precise. I'm not sure if it all sounds as good, though : /

### CODE REVIEW

**Comment by mdhaber** [comment]
I intended to create a new branch for the rest of these intro edits : / Oops.

File: scipy-1.0/paper.tex [hunk 1]
```diff
@@ -153,22 +153,24 @@ \section*{Introduction}
...
-interpolation, eigenvalues, and linear systems.  It also provides
+language that provides fundamental building blocks for modeling and
+solving scientific problems. 
+SciPy includes algorithms for optimization, integration, interpolation,
+eigenvalue problems, algebraic equations, differential equations, and many other
+classes of problems; it also provides
 specialized data structures, such as sparse arrays and k-dimensional
 trees. SciPy is built on top of NumPy\cite{vanderwalt2011numpy,Oliphant-2015},
-which provides array data structures and related fast numerical routines.
+which provides array data structures and related fast numerical routines, and
```
[the remark this answers is not in the document]
Comment (tylerjereddy) [reply scipy-1.0/paper.tex]: Not worth the energy to change it from original, but also not worth changing it back now, so I'll mark as resolved.

File: scipy-1.0/paper.tex [hunk 2]
```diff
@@ -153,27 +153,29 @@ \section*{Introduction}
...
+scripts\cite{alex_nitz_2018_1596771, LIGO-open}
+used in the analysis of gravitational waves
+\cite{PhysRevD.93.122003, abbott2017gw170817} 
+import several submodules of SciPy.
+
 
 Recently, SciPy released version 1.0, a milestone that traditionally
 signals a library's API (Applications Programming Interface) being
-mature enough to be be trusted in production pipelines.  The SciPy 1.0
-version numbering, however, belies the history of a project that has
```
[the remark this answers is not in the document]
Comment (tylerjereddy) [reply scipy-1.0/paper.tex]: I prefer the explicit usage of 1.0 over the usage of "this," which can be confusing to readers, but again not worth the energy to change or reverse

File: scipy-1.0/paper.tex [hunk 3]
```diff
@@ -188,7 +190,7 @@ \section*{Introduction}
 
 Yet, the philosophical motivations behind a fully open toolstack, and
 an excited, friendly community with a singular focus, seemed to have
-proven decisive in the long run.  It led not only to the library
```
[the remark this answers is not in the document]
Comment (tylerjereddy) [reply scipy-1.0/paper.tex]: I don't think this debate is worth core dev time -- let's just use your wording & move on

File: scipy-1.0/paper.tex [hunk 4]
```diff
@@ -197,16 +199,16 @@ \section*{Introduction}
 better practices---both for community management and software
 development.
 
-Despite its success, SciPy still faces challenges:
```
[the remark this answers is not in the document]
Comment (tylerjereddy) [reply scipy-1.0/paper.tex]: Agreed

File: scipy-1.0/paper.tex [hunk 5]
```diff
@@ -218,18 +220,17 @@ \section*{Background}
...
 libraries and to then drive those libraries interactively.  Scientists
 could thereby gain access to a wide variety of existing computational
-libraries, without concerning themselves with low-level programming
+libraries without concerning themselves with low-level programming
 concepts such as memory management.
 
-In 1995, Jim Hugunin, a graduate student from MIT, wrote in the first
-message on the Python Matrix Special Interest Group (Matrix-SIG)
+In 1995, Jim Hugunin, a graduate student from MIT, wrote the first
+message in a new Python Matrix Special Interest Group (Matrix-SIG)
```
[the remark this answers is not in the document]
Comment (tylerjereddy) [reply scipy-1.0/paper.tex]: I'm happy with your wording -- I don't think the debate here is too important -- it was "early" / seminal message of some sort.

File: scipy-1.0/paper.tex [hunk 6]
```diff
@@ -255,15 +256,14 @@ \section*{Background}
...
-computing, here scientific features are added to a general purpose language
-instead.  This broadens the scope of problems that can be addressed easily, the
+computing, here scientific features are added to a general purpose language.  
+This broadens the scope of problems that can be addressed easily, expands the
 sources of data that are readily accessible, and increases the size of the
-community that develops general purpose tooling.
+community that develops code for the platform.
 
 The availability of a standard numerical array data structure in
-Python led to a sudden and rapidly growing number of scientific
```
[the remark this answers is not in the document]
Comment (tylerjereddy) [reply scipy-1.0/paper.tex]: looks fine before & after to me so not worth debate!

File: scipy-1.0/references.bib [hunk 7]
```diff
@@ -1529,6 +1529,17 @@ @article{LIGO-open
   year={2015}
 }
 
+@article{abbott2017gw170817,
```
[the remark this answers is not in the document]
Comment (tylerjereddy) [reply scipy-1.0/references.bib]: If this makes sense to you let's just go with it.

**Comment by tylerjereddy** [comment]
Thanks Matt

**Pull request merged**